oaknational/Oak-Web-Application
# Feat/pupil 1054/update avo tracking properties

## Description

Music year: 1978

1. pupil video events (pause, play etc)
    - pupil videos use same event as e.g. blog video so in order to send pupil specific data lessons, units, etc we are making those properties optional video event properties
2. lesson completed/ started
    - added phase and year properties
    - added `activity time spent` field to lesson events
3. lesson summary reviewed
4. share/ printable results clicked

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2887--oak-web-application.netlify.thenational.academy
2. Click on a lesson
3. You should see video events pause, play etc will return lesson specific data
4. you should see lesson completed/ started fired with phase and year properties
5. added `activity time spent` field to lesson events to activities when they are completed and abandoned
6. lesson summary event called when review page is visited
7. share event fired when share button is selected

### Acceptance criteria

- [ ]  pupil video events correct properties
- [ ]  lesson completed/ started correct properties
- [ ]  lesson summary reviewed correct properties
- [ ]  send events when share is selected


## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
